source/features/forked-to.tsx

```diff
@@ -122,12 +116,9 @@ async function init(): Promise<void | false> {
 		await updateUI(forks);
 	}
 
-	// This feature only applies to users that have multiple organizations, because that makes a fork picker modal appear when clicking on "Fork"
-	const hasOrganizations = await elementReady('details-dialog[src*="/fork"]', {waitForChildren: false});
-
 	// Only fetch/update forks when we see a fork (on the current page or in the cache).
 	// This avoids having to `updateCache` for every single repo you visit.
-	if (forks || (hasOrganizations && pageDetect.isForkedRepo())) {
+	if (forks || pageDetect.isForkedRepo()) {
```

Comment:
Can you flip this condition to allow an early return? I think

```suggestion
	if (!forks && !pageDetect.isForkedRepo()) {
		return false;
	}
```